install_script_opw.sh

```diff
@@ -0,0 +1,644 @@
+        agent_version_custom="$(apt-cache madison observability-pipelines-worker | grep -E "$pkg_pattern" -om1)" || true
+        verify_agent_version "="
+    fi
+
+    declare -a packages
+    packages=("observability-pipelines-worker" "datadog-signing-keys")
+
+    echo -e "  \033[33mInstalling package(s): ${packages[*]}\n\033[0m"
+
+    $sudo_cmd apt-get install -y --force-yes "${packages[@]}"
+
+    ERROR_MESSAGE=""
+else
+    ERROR_MESSAGE="Your OS or distribution are not supported by this install script.
+Please follow the instructions on the Observability Pipelines Worker setup page:
+
+https://docs.datadoghq.com/observability_pipelines/setup/"
+    ERROR_CODE=$UNSUPPORTED_PLATFORM_CODE
+    printf "\033[31m$ERROR_MESSAGE\033[0m\n"
+    report_telemetry
+    exit;
+fi
+
+# Set the configuration
+if [ "$apikey" ]; then
+  printf "\033[34m\n* Adding your API key to the $nice_flavor configuration: $config_file\n\033[0m\n"
+  $sudo_cmd sh -c "sed -i 's/# api_key:.*/api_key: $apikey/' $config_file"
+else
+  # If the import script failed for any reason, we might end here also in case
+  # of upgrade, let's not start the worker or it would fail because the api key
+  # is missing
+  if ! $sudo_cmd grep -q -E '^api_key: .+' "$config_file"; then
+    printf "\033[31mThe $nice_flavor won't start automatically at the end of the script because the API key is missing,\nplease add one in $config_file and start the $nice_flavor manually.\n\033[0m\n"
+    no_start=true
+  fi
+fi
+
+if [ "$op_pipeline_id" ]; then
+  printf "\033[34m\n* Adding your Pipeline ID to the $nice_flavor configuration: $config_file\n\033[0m\n"
+  $sudo_cmd sh -c "sed -i 's/# pipeline_id:.*/pipeline_id: $op_pipeline_id/' $config_file"
+else
+  # If the import script failed for any reason, we might end here also in case
+  # of upgrade, let's not start the worker or it would fail because the api key
+  # is missing
+  if ! $sudo_cmd grep -q -E '^pipeline_id: .+' "$config_file"; then
+    printf "\033[31mThe $nice_flavor won't start automatically at the end of the script because the Pipeline ID is missing,\nplease add one in $config_file and start the $nice_flavor manually.\n\033[0m\n"
+    no_start=true
+  fi
+fi
+
+if [ "$site" ]; then
+  printf "\033[34m\n* Setting SITE in the $nice_flavor configuration: $config_file\n\033[0m\n"
+  $sudo_cmd sh -c "sed -i 's/# site:.*/site: $site/' $config_file"
+fi
+
+if [ ! -e "$etcdir"/pipeline.yaml ]; then
+  printf "\033[31mThe $nice_flavor won't start automatically at the end of the script because the pipeline configuration is missing,\nplease add one at $etcdir/pipeline.yaml and start the $nice_flavor manually.\n\033[0m\n"
+  no_start=true
+fi
+
+$sudo_cmd chown observability-pipelines-worker:observability-pipelines-worker "$config_file"
+$sudo_cmd chmod 640 "$config_file"
+
+# Creating or overriding the install information
+install_info_content="---
```

Comment:
The install_info is something that only the Agent uses and submits it with the metadata payload, so we don't really need it here. OTOH, we will probably eventually need something like it for OPW as well, so I agree with keeping it.

Reply:
Yep, it seemed like a "would be useful in the future, and easy enough to adjust here" so I opted to keep it in.